docs/tutorials/scenario-simulation/.pages

```diff
@@ -1,5 +1,5 @@
 nav:
   - index.md
-  - planning-simulation.md
-  - rosbag-replay-simulation.md
-  - digital-twin-simulation.md
+  - installation.md
+  - scenario-test-simulation.md
+  - random-test-simulation.md
```

Comment:
@WJaworskiRobotec Could you add files under the `planning-simulation/` directory, and keep `rosbag-replay-simulation.md` and `digital-twin-simulation.md` as-is?

Reply:
so you mean to have docs/tutorials/scenario-simulation/planning-simulation/(scenario/random)-test-simulation.md ?

Reply:
Exactly. It's because we'll add scenario testing with the digital-twin simulator in the future.

Reply:
Since it was a bit incorrect, I've fixed `.pages`. :bow:  e322fb533b37e9fd7877c0a3eb0884dc3aafd4fa

Reply:
Thanks a lot !